src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```diff
@@ -28,6 +30,15 @@ export class ActionBarOverflowMenu {
     @Prop({ reflect: true })
     public openDirection: OpenDirection = 'bottom-end';
 
+    @Prop()
+    public actionBarIsShrunk = false;
```

Comment:
To make this even more generic, could you instead add a `Icon` to the interface?
```suggestion
    @Prop()
    public overFlowIcon?: Icon;
```

Reply:
I think we don't need to pass the icon from the parent component since the icon is hard coded. I added my suggestion [here](https://github.com/Lundalogik/lime-elements/pull/3511#discussion_r2018526558)

Reply:
I went with Pelle's approach here.  
@civing Could you check Bef's comment here.